### DISCUSSION TITLE
App.Support.QuickAssist

### DISCUSSION THREAD

**Discussion by cespedesdan** [farag2/Sophia-Script-for-Windows#112] [Q&A] [answered]
Could anyone help me to get **App.Support.QuickAssist** back to the system?

**Comment by farag2** [comment, marked as answer]
Hello! You can restore it via PowerShell (internet connection required):

```powershell
Get-WindowsCapability -Online | Where-Object -FilterScript {$_.Name -cmatch "App.Support.QuickAssist*"} | Add-WindowsCapability -Online
```

Manually (internet connection required):
```Open Settings - Apps - Manage optional features - Enable "Microsoft Quick Assist"```

Did you try to run ```WindowsCapabilities -Enable``` to restore?

**Reply by cespedesdan** [reply]
Thank you so much, ``` WindowsCapabilities -Enable ``` didn't work in my first attemp.

**Reply by farag2** [reply]
Log error?

**Reply by cespedesdan** [reply]
I don't save the log

**Reply by farag2** [reply]
What did the console output on error? Did the module load normal?
